main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	"github.com/gin-gonic/gin"
+	"github.com/joho/godotenv"
+	"github.com/okkur/reposeed-server/generator"
+	"github.com/okkur/reposeed/cmd/reposeed/config"
+)
+
+func main() {
+	err := godotenv.Load()
+	if err != nil {
+		log.Println("Couldn't find .env file. Reading environment variables from system")
+	}
+	app := gin.Default()
+	config := &config.Config{}
+	app.POST("/generate", func(ctx *gin.Context) {
+		ctx.BindJSON(config)
+		filename := generator.CreateFiles(*config, "../templates", config.Project.Name, os.Getenv("STORAGE"))
```

Comment:
I guess CreateFiles could also return an error so that we can print it to the user, if something is wrong with their json file ?